syntax = "proto3";
package streetgangpb;


message GetVersionResponse
{
	string track_id = 1;
	int32 result = 2;
	string version = 3;
}
